### ISSUE TITLE
Form group label - font-size

### ISSUE THREAD

**Issue by tonifisler** [epfl-idevelop/elements#136]
The font-size of the form-group labels is a bit too small in my opinion. It should be the same as the body font-size, no? 13px seems a bit small for such information. `/#/molecules/form-group`

![screen shot 2018-06-11 at 14 30 24](https://user-images.githubusercontent.com/1708450/41231636-045ab740-6d84-11e8-891b-112ebe4ea03f.png)
